src/read/xcoff/section.rs

```diff
@@ -298,12 +306,24 @@ pub trait SectionHeader: Debug + Pod {
             Ok(&[])
         }
     }
+
+    /// Read the relocations in a XCOFF file.
+    ///
+    /// `data` must be the entire file data.
+    fn xcoff_relocations<'data, R: ReadRef<'data>>(
```

Comment:
```suggestion
    fn relocations<'data, R: ReadRef<'data>>(
```

You're probably basing your code on COFF. However, it does that because it needs to distinguish between COFF and PE, but there's no need for that here.